app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -117,6 +118,7 @@ object PatientMocker {
       pinCode: String? = null,
       protocolUuid: UUID? = UUID.randomUUID(),
       groupUuid: UUID? = UUID.randomUUID(),
+      location: Coordinates = Coordinates(latitude = 1.908537, longitude = 73.537524),
```

Comment:
This property should be nullable, right?

Reply:
Yep, fixing this